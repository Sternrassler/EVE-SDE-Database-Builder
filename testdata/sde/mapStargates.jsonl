{"stargateID":50000001,"solarSystemID":30000001,"destination":50000002,"typeID":29624,"x":1.0e12,"y":0.0,"z":0.0}
{"stargateID":50000002,"solarSystemID":30000142,"destination":50000001,"typeID":29624,"x":-1.0e12,"y":0.0,"z":0.0}
{"stargateID":50000003,"solarSystemID":30000003,"destination":50000004,"typeID":29624,"x":5.0e11,"y":1.0e11,"z":-2.0e11}
