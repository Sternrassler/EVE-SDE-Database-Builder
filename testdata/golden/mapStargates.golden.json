[
  {
    "stargateID": 50000001,
    "solarSystemID": 30000001,
    "destinationID": null
  },
  {
    "stargateID": 50000002,
    "solarSystemID": 30000142,
    "destinationID": null
  },
  {
    "stargateID": 50000003,
    "solarSystemID": 30000003,
    "destinationID": null
  }
]